armotypes/notificationreporttypes.go

```diff
@@ -0,0 +1,151 @@
+package armotypes
+
+type WeeklyReport struct {
+	ClustersScannedThisWeek             int                 `json:"clustersScannedThisWeek" bson:"clustersScannedThisWeek"`
+	ClustersScannedPrevWeek             int                 `json:"clustersScannedPrevWeek" bson:"clustersScannedPrevWeek"`
+	LinkToConfigurationScanningFiltered string              `json:"linkToConfigurationScanningFiltered" bson:"linkToConfigurationScanningFiltered"`
+	RepositoriesScannedThisWeek         int                 `json:"repositoriesScannedThisWeek" bson:"repositoriesScannedThisWeek"`
+	RepositoriesScannedPrevWeek         int                 `json:"repositoriesScannedPrevWeek" bson:"repositoriesScannedPrevWeek"`
+	LinkToRepositoriesScanningFiltered  string              `json:"linkToRepositoriesScanningFiltered" bson:"linkToRepositoriesScanningFiltered"`
+	RegistriesScannedThisWeek           int                 `json:"registriesScannedThisWeek" bson:"registriesScannedThisWeek"`
+	RegistriesScannedPrevWeek           int                 `json:"registriesScannedPrevWeek" bson:"registriesScannedPrevWeek"`
+	LinkToRegistriesScanningFiltered    string              `json:"linkToRegistriesScanningFiltered" bson:"linkToRegistriesScanningFiltered"`
+	Top5FailedControls                  []TopCtrlItem       `json:"top5FailedControls" bson:"top5FailedControls"`
+	Top5FailedCVEs                      []TopVulItem        `json:"top5FailedCVEs" bson:"top5FailedCVEs"`
+	ClustersScanned                     []ClusterScanned    `json:"clustersScanned" bson:"clustersScanned"`
+	RepositoriesScanned                 []RepositoryScanned `json:"repositoriesScanned" bson:"repositoriesScanned"`
+	RegistriesScanned                   []RegistryScanned   `json:"registriesScanned" bson:"registriesScanned"`
+}
+
+type ClusterScanned struct {
+	Cluster ClusterResourceScanned `json:"cluster" bson:"cluster"`
+}
+
+type RegistryScanned struct {
+	Registry ResourceScanned `json:"registry" bson:"registry"`
+}
+
+type RepositoryScanned struct {
+	Repository ResourceScanned `json:"repository" bson:"repository"`
+}
+
+type ClusterResourceScanned struct {
```

Comment:
ClusterResourceScanned  can use ResourceScanned as base struct and just add  shortName
(```fullName'``will be ``name ``)